src/modules/socketClient/SocketClient.ts

```diff
@@ -0,0 +1,42 @@
+import { io, Socket } from "socket.io-client";
+
+class SocketClient {
+
+  socket: Socket
+  isSocketConnected: boolean
+  
+  constructor() {
+    this.isSocketConnected = false;
+    this.socket = io("http://localhost:5000");
+    
+    this.checkSocketConnection().then(() => {
+      this.isSocketConnected = this.socket.connected
+    })
+  }
+
+  async checkSocketConnection(): Promise<any>{
```

Comment:
Can we avoid the `any` here

Reply:
I tried Promise<T>, Promise<boolean> (the return type of the object getting resolved) but the editor gives me a linting error.  

I am thinking to go with Promise<void>? would that work?

Reply:
Yes `Promise<void>` will do